### PR TITLE
TM-5497-AIM-Remarks

### PR DESCRIPTION
**Pull request by cagbakwu**
[Ticket](https://metaphase.atlassian.net/browse/TM-5497)

### CODE REVIEW

File: src/Components/Agenda/AgendaItemMaintenanceContainer/AgendaItemMaintenanceContainer.jsx [hunk 1]
```diff
@@ -134,15 +134,17 @@ const AgendaItemMaintenanceContainer = (props) => {
...
-      remark$.user_remark_inserts = [];
-      remark$.remark_inserts.forEach(ri => (remark$.user_remark_inserts.push({
-        airiinsertiontext: textInputs[ri.rirmrkseqnum][ri.riseqnum],
-        airirmrkseqnum: ri.rirmrkseqnum,
-        aiririseqnum: ri.riseqnum,
-      })));
-
-      userRemarks$.push(remark$);
-      setUserRemarks(userRemarks$);
+      if (Object.keys(remark$).length !== 0) {
```
Comment (cagbakwu) [inline src/Components/Agenda/AgendaItemMaintenanceContainer/AgendaItemMaintenanceContainer.jsx:137]: If the potential remarks are empty, it just returns the original set remarks if any.

**Review by M00staff — CHANGES_REQUESTED** [review]
I might be off here - but, I cannot recreate the issue from the ticket

It looks to me like text needs to be entered in order for the user to be able to save a remark?

image

It would be nice if the ticket had some kind of steps to be able to reproduce - but, also it looks like the ticket was created in January ... could it be that this was already solved?

File: src/Components/Agenda/AgendaItemMaintenanceContainer/AgendaItemMaintenanceContainer.jsx (hunk 1 above)
Comment (M00staff) [reply src/Components/Agenda/AgendaItemMaintenanceContainer/AgendaItemMaintenanceContainer.jsx:137]: I'm not sure what this is even doing ... if you `console.log(Object.keys(remark$));` this is what you get, for any added remark. 

image

it doesn't have anything to do with the remark insert

**Comment by M00staff** [comment]
yea - it looks to me like this issue was solved by this PR, about 3 months ago

https://github.com/MetaPhase-Consulting/State-TalentMAP/pull/2868

ticket - https://metaphase.atlassian.net/browse/TM-5556 

... did you even check to see if this was an issue before trying to solve it?

Unless, I'm missing something?

**Comment by cagbakwu** [comment]
@cagbakwu yea the text had to be entered for the save to become available but I was just adding a second check if somehow the app still allowed a save without it. It should never get to this so I could just close this one out.

**Comment by cagbakwu** [comment]
@M00staff I did see that it wouldn't save without the text but I added a second check just in case the app bypassed the check. I didn't realize the creation date of this task until now. Ill close this one out.

**Pull request closed without merging**